xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/layouts/table/LayoutTableHeaderNames.vue

```diff
@@ -72,23 +72,19 @@
           :title="$t('livedata.action.reorder.hint')"
           @click.stop
         >
-          <span class="fa fa-ellipsis-v"></span>
+          <!-- TODO: more-vertical and caret-down as the same icon in Silk. -->
+          <XWikiIcon :icon-descriptor="{name: 'more-vertical'}"/>
         </div>
         <!-- Property Name -->
         <span class="property-name">{{ property.name }}</span>
         <!--
           Sort icon
           Only show the icon for the first-level sort property
         -->
-        <span
+        <XWikiIcon
           v-if="logic.isPropertySortable(property.id)"
-          :class="[
-            'sort-icon',
-            'fa',
-            { 'sorted': isFirstSortLevel(property) },
-            (isFirstSortLevel(property) && firstSortLevel.descending) ? 'fa-caret-up' : 'fa-caret-down',
-          ]"
-        ></span>
+          :icon-descriptor="{name: isFirstSortLevel(property) && firstSortLevel.descending? 'caret-up': 'caret-down'}"
+          :additional-classes="['sort-icon',  isFirstSortLevel(property)? 'sorted': '']"/>
```

Comment:
Why do you need here to create an `additional-classes` property on the XWikiIcon component? Can't you simply use the regular `class` attribute here?

Reply:
I yes, I forgot classes of a component call are passed to the component root. Indeed that would be simple and more idiomatic that way. I'll fix that.